From: Markus Koschany <apo@gambaru.de>
Date: Fri, 5 Oct 2012 15:16:04 +0200
Subject: Improve-byzanz-record-manpage
Forwarded: https://bugzilla.gnome.org/show_bug.cgi?id=685308

Applied the old patch from Jari Aalto again which was accidentally removed
by one of the last NMUs. Added entry about webm support.

---
 src/byzanz-record.1 |   61 ++++++++++++++++++++++++++++-----------------------
 1 file changed, 33 insertions(+), 28 deletions(-)

diff --git a/src/byzanz-record.1 b/src/byzanz-record.1
index ea935e7..71087db 100644
--- a/src/byzanz-record.1
+++ b/src/byzanz-record.1
@@ -28,51 +28,60 @@ to use the panel applet instead.
 .SH OPTIONS
 .SS "Application Options:"
 .TP
+\fB\-a\fR, \fB\-\-audio\fR
+Record audio from the default input device. This only works if the output format
+supports it and will otherwise cause an error.
+.TP
+\fB\-c\fR, \fB\-\-cursor\fR
+Record mouse cursor
+.TP
 \fB\-d\fR, \fB\-\-duration\fR=\fISECS\fR
 Duration of animation (default: 10 seconds)
 .TP
 \fB\-\-delay\fR=\fISECS\fR
 Delay before start (default: 1 second)
 .TP
-\fB\-c\fR, \fB\-\-cursor\fR
-Record mouse cursor
-.TP
-\fB\-a\fR, \fB\-\-audio\fR
-Record audio from the default input device. This only works if the output format
-supports it and will otherwise cause an error.
+\fB\-\-display\fR=\fIDISPLAY\fR
+X display to use
 .TP
-\fB\-x\fR, \fB\-\-x\fR=\fIPIXEL\fR
-X coordinate of rectangle to record
+\fB\-h\fR, \fB\-\-height\fR=\fIPIXEL\fR
+Height of recording rectangle
 .TP
-\fB\-y\fR, \fB\-\-y\fR=\fIPIXEL\fR
-Y coordinate of rectangle to record
+\fB\-v\fR, \fB\-\-verbose\fR
+be verbose
 .TP
 \fB\-w\fR, \fB\-\-width\fR=\fIPIXEL\fR
 Width of recording rectangle
 .TP
-\fB\-h\fR, \fB\-\-height\fR=\fIPIXEL\fR
-Height of recording rectangle
-.TP
-\fB\-v\fR, \fB\-\-verbose\fR
-Be verbose
+\fB\-x\fR, \fB\-\-x\fR=\fIPIXEL\fR
+X coordinate of rectangle to record
 .TP
-\fB\-\-display\fR=\fIDISPLAY\fR
-X display to use
+\fB\-y\fR, \fB\-\-y\fR=\fIPIXEL\fR
+Y coordinate of rectangle to record
 .SS "Help Options:"
 .TP
 \-?, \fB\-\-help\fR
 Show help options
 .TP
-\fB\-\-help\-gtk\fR
-Show GTK+ Options
-.TP
 \fB\-\-help\-all\fR
 Show all help options
+.TP
+\fB\-\-help\-gtk\fR
+Show GTK+ Options
 .SH OUTPUT FILE
 After \fBbyzanz-record\fP is finished, the recording is written to FILENAME.
 The format is determined by the filename extension. The following formats are
 supported:
 .TP
+\fBbyzanz\fR
+Record to Byzanz' internal debugging format. This is useful for benchmarking
+Byzanz or if you want to convert the recording to multiple formats later.
+You can use \fBbyzanz-playback\fP(1) to convert the file.
+.TP
+\fBflv\fR
+Record to a Flash Screen video. This recording method is lossless. Use it if
+you want to postprocess the file in other applications.
+.TP
 \fBgif\fR
 Record to an animated GIF image. Use this if you want to record a mostly
 static screen with a limited amount of colors, such as using a file manager
@@ -83,14 +92,10 @@ This is the default and will be used if an unrecognized extension is used.
 Record to an Ogg Theora video. This format supports audio. Use this if you
 want to record dynamic contents, such as video playback.
 .TP
-\fBflv\fR
-Record to a Flash Screen video. This recording method is lossless. Use it if
-you want to postprocess the file in other applications.
-.TP
-\fBbyzanz\fR
-Record to Byzanz' internal debugging format. This is useful for benchmarking
-Byzanz or if you want to convert the recording to multiple formats later.
-You can use \fBbyzanz-playback\fP(1) to convert the file.
+\fBwebm\fR
+Record to a WebM video. This format consists of VP8 video and Vorbis audio
+streams. Use this if you want to record dynamic contents, e.g. for a HTML5
+video.
 .SH SEE ALSO
 \fBbyzanz-playback\fR(1)
 .SH AUTHOR
